packages/checkbox-group/test/lion-checkbox-indeterminate.test.js

```diff
@@ -301,18 +331,20 @@ describe('<lion-checkbox-indeterminate>', () => {
     const elParentIndeterminate = /**  @type {LionCheckboxIndeterminate} */ (el.querySelector(
       '#parent-checkbox-indeterminate',
     ));
+    const ElNestedSubCheckboxes = getProtectedMembers(elNestedIndeterminate);
+    const ElParentSubCheckboxes = getProtectedMembers(elParentIndeterminate);
 
     // Act - check a nested checkbox
-    elNestedIndeterminate?._subCheckboxes[0]._inputNode.click();
+    if (elNestedIndeterminate) ElNestedSubCheckboxes.subCheckboxes[0]._inputNode.click();
```

Comment:
We have the convention to always use parentheses and newlines for an if for readability (should be captured by linting though...)